package.json

```diff
@@ -36,7 +36,7 @@
     "lint-shared": "cd shared && yarn run lint",
     "lint-root": "cd root && yarn run lint",
     "lint": "yarn lint-legacy && yarn lint-ui && yarn lint-shared && yarn lint-root",
-    "test-cypress": "cd proxy && yarn run cypress-ci",
+    "test-cypress": "yarn --cwd integration run cypress-test",
```

Comment:
There are a couple of other cypress commands in this file that need updating too.